frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -402,8 +424,21 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
                 isRequired
                 id={asId('url')}
                 value={url}
-                onChange={(v) => setURL(v)}
+                onChange={setURL}
               />
+              <div className="pf-c-form__helper-text" aria-live="polite">
```

Comment:
can you please extract this to a simple component?

Reply:
done, did the same for container source help

Reply:
+1